qcore/constants.py

```diff
@@ -353,6 +353,7 @@ class Components(ExtendedStrEnum):
     crotd50 = 4, "rotd50"
     crotd100 = 5, "rotd100"
     crotd100_50 = 6, "rotd100_50"
+    cnorm = 7, "norm"
 
     @staticmethod
     def get_comps_to_calc_and_store(arg_comps: List[str]):
```

Comment:
Is the logic for get_comps_to_calc_and_store still valid with the norm component?
Sounds similar to geom, so probably ok?

Reply:
Yep should be fine. Norm is generated from Adv_IM so it's not a 'basic' component.